### DISCUSSION TITLE
Go into fullscreen on app startup

### DISCUSSION THREAD

**Discussion by lospeq** [microsoft/WindowsAppSDK#1881] [Q&A]
I can't seem to get the app to go into fullscreen mode with the following code. The title is set correctly and logs show that the presenter does indeed change, but the app stays windowed. 
```
 public MainWindow()
        {
            this.InitializeComponent();

            IntPtr hWnd = WindowNative.GetWindowHandle(this);
            WindowId myWndId = Win32Interop.GetWindowIdFromWindow(hWnd);
            AppWindow appWindow = AppWindow.GetFromWindowId(myWndId);

            if (appWindow != null)
            {
                Debug.WriteLine(appWindow.Presenter); // Microsoft.UI.Windowing.OverlappedPresenter
                appWindow.SetPresenter(AppWindowPresenterKind.FullScreen);
                appWindow.Title = "Test";
                Debug.WriteLine(appWindow.Presenter); // Microsoft.UI.Windowing.FullScreenPresenter
            }
        }
```
The same code works just fine if called somewhere later in the app and not inside of window initialization. So basically presenter change must be called only after Activate() was called on the window?

Also not sure if this is proper behavior, but after the app goes into fullscreen mode and the window title bar is hidden it usually can be shown again by bringing the cursor to the screen top. Here, however, after being hidden once the title bar does not appear anymore.

**Comment by castorix** [comment]
You must call :
```
appWindow.Show(true);
appWindow.SetPresenter(AppWindowPresenterKind.FullScreen);
```

And I cannot reproduce what you say with "_it usually can be shown again by bringing the cursor to the screen top"_

**Reply by lospeq** [reply]
In uwp app when in fullscreen mode you can move your cursor up to the very top to make the title bar visible again and, say, press the exit button. After porting the app to windows app sdk this behaviour does not happen.
